.github/api_guidelines.md

```diff
@@ -8,10 +8,130 @@ in slack channel [#forum-acs-api-design](https://redhat-internal.slack.com/archi
 ## Table of Contents
 
 - [Naming Guidelines](#naming-guidelines)
+  - [Decoupling](decoupling)
```

Comment:
It looks like the link is broken. Did you mean `  - [Decoupling](#decoupling)` ?

Reply:
good catch